static/settings.config.php

```diff
@@ -52,10 +52,6 @@
 		// Enter 0 for no time limit.
 		'account_abandon_days' => 0,
 
-		// addon (Comma-separated list)
-		// Manual list of addons which are enabled on this system.
-		'addon' => '',
```

Comment:
Don't forget to remove the mentions to this configuration key in the various addon install documentation.

Reply:
thx: https://github.com/friendica/friendica-addons/pull/1345